gapir/cc/memory_manager.cpp

```diff
@@ -75,19 +75,36 @@ MemoryManager::MemoryManager(const std::vector<uint32_t>& sizeList) : mConstantM
     }
 
     GAPID_DEBUG("Base address: %p", mMemory.get());
-    setReplayDataSize(0);
+    setReplayDataSize(0, 0);
     setVolatileMemory(mSize);
 }
 
-bool MemoryManager::setReplayDataSize(uint32_t size) {
-    if (size > mSize) {
-        return false;
-    }
-
-    mReplayData = {align(mMemory.get() + mSize - size), size};
-    GAPID_DEBUG("Replay range: [%p,%p]",
-        mReplayData.base, mReplayData.base + mReplayData.size - 1);
-    return true;
+bool MemoryManager::setReplayDataSize(uint32_t constantMemorySize,
```

Comment:
setReplayDataSize() is changed to accept two parameters. And there won't be a setConstantMemory() anymore